_posts/2023-04-11 natto.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'I wrote a book on Natto '
+image: assets/images/natto_book/cover.jpg
+tags:
+- natto
+- solid-state-fermentation
+categories:
+- industry
+author: keith
+---
+
+It was fun to write. Here are a few thoughts on natto, writing, and academia...
+
+### There were no detailed books on Natto
+
+In 2022, if you searched 'natto book' on Amazon.com, the first result is a self-help / insomia book: something about a problem getting stickier the more you stir it. I'm sure it's a great book, but... come on. 
+
+Aside from that, there are a few decent home fermentation books with passing mentions about natto. Kristen and Christopher Shockey's book is great, and so is Sandor Katz's. Natto chapters also show up in a few textbooks on fermented foods, but none of them have the sort of detail I'm looking for. 
+
+What kind of book did I want? Something like this:
+
+![](/assets/images/natto-book/kojistudies_book.jpg){:width="550" .center}
```

Comment:
@daizu27 it's the name of the path, too: `natto-book` vs `natto_book`. I'll make the fix and deploy